apps/api/v2/src/ee/event-types/event-types_2024_06_14/controllers/event-types.controller.e2e-spec.ts

```diff
@@ -27,6 +27,7 @@ import {
   UpdateEventTypeInput_2024_06_14,
 } from "@calcom/platform-types";
 import { BookingWindowPeriodInputTypeEnum_2024_06_14 } from "@calcom/platform-types/dist/event-types/event-types_2024_06_14/inputs/enums/booking-window.enum";
+import { FrequencyInput } from "@calcom/platform-types/dist/event-types/event-types_2024_06_14/inputs/enums/frequency";
```

Comment:
please don't import from /dist folder, export the file inside of platform/types/index.js and import from "@calcom/platform-types" after rebuilding it